alpha/apps/kaltura/lib/kUploadTokenMgr.php

```diff
@@ -341,10 +341,17 @@ protected function handleResume($fileData, $resumeAt)
 		return $currentFileSize;
 	}
 	
-	private function appendCurrentChunk($targetFilePath, $chunkFilePath)
+	private function appendCurrentChunk($targetFilePath, $chunkFilePath, $resumeAt)
 	{
 		$targetFileResource = fopen($targetFilePath, 'r+b');
-		fseek($targetFileResource, 0, SEEK_END);
+		if($resumeAt == -1)
```

Comment:
If resume at == -1 is an application assumption of last chunk, than it should be defined in the class (using const or something)